easypost/__init__.py

```diff
@@ -321,27 +329,40 @@ def interpret_response(self, http_body, http_status):
             self.handle_api_error(http_status, http_body, response)
         return response
 
-    def requests_request(self, method, abs_url, headers, params):
+    def requests_request(self, method, abs_url, headers, params, isRating):
         method = method.lower()
         if method == 'get' or method == 'delete':
             if params:
                 abs_url = self.build_url(abs_url, params)
             data = None
         elif method == 'post' or method == 'put':
-            data = self.encode(params)
+            if not isRating:
```

Comment:
This seems pretty hacky. It seems like the check that you want here is if `headers['Content-Type'] == 'application/json'` (beware the capital T as mentioned above) vs passing `isRating` into this method.